cpplite/cpplite.editor/src/org/netbeans/modules/cpplite/editor/lsp/LanguageServerImpl.java

```diff
@@ -64,9 +64,10 @@
 })
 public class LanguageServerImpl implements LanguageServerProvider {
 
+    private static final boolean DEBUG = Boolean.getBoolean("cpplite.lsp.debug");
```

Comment:
The best we could do here is LOG.isLoggable(FINE) or alike instead of DEBUG - we need to print exactly what is in the streams (as often that shows some important piece of information), which, as far as I know, is not an ideal usecase for Loggers. (To use Loggers, we would need to send the information to the Logger in "batches", e.g. by lines, but that is often troublesome with this protocol.)

Reply:
`LOG.isLoggable(FINE)` would be more standard - at least it would be known how to configure it. If you insist on the property, document it in `arch.xml` as `<api group="systemproperty"/>`.

Reply:
Done.